uniffi_bindgen/src/lib.rs

```diff
@@ -97,84 +97,68 @@ const BINDGEN_VERSION: &str = env!("CARGO_PKG_VERSION");
 use anyhow::{anyhow, bail, Context, Result};
 use camino::{Utf8Path, Utf8PathBuf};
 use fs_err::{self as fs, File};
-use serde::{Deserialize, Serialize};
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
```

Comment:
(can't comment on unchanged lines)

Line 75 above still uses the `include!(concat!(...))` part in the docs.
This should also be changed to the `uniffi::include_scaffolding!` usage.